app/uk/gov/hmrc/agentsubscriptionfrontend/models/AgentSession.scala

```diff
@@ -32,7 +32,9 @@ case class AgentSession(
   registration: Option[Registration] = None,
   checkAmls: Option[String] = None,
   amlsAppliedFor: Option[String] = None,
-  amlsDetails: Option[AMLSDetails] = None)
+  amlsDetails: Option[AMLSDetails] = None,
+  identifyBusinessTaskComplete: Boolean = false,
```

Comment:
can we use short name here: `businessTaskComplete`

Reply:
sure